firebase/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "memo-firebase",
   "engines": {
-    "node": "14"
+    "node": "16"
```

Comment:
Update Node version to latest stable version to use the same version we use in our development machines and also because node `14` has some issues with `postinstall` script, specially in CI's